packages/gooddata-sdk/tests/catalog/unit_tests/test_notification_channel_destinations.py

```
@@ -0,0 +1,133 @@
+from gooddata_api_client.model.notification_channel_destination import NotificationChannelDestination
+from gooddata_sdk import (
+    CatalogDeclarativeNotificationChannel,
+    CatalogDefaultSmtp,
+    CatalogInPlatform,
+    CatalogSmtp,
+    CatalogWebhook,
+)
+
+DESTINATIONS = {
+    "WEBHOOK": (
+        CatalogWebhook(url="https://webhook.site/hook", token="secret"),
```

Ruff (0.16.2)

[error] Possible hardcoded password assigned to argument: "token"

(S106)